samples/cpp/visual_language_chat/visual_language_chat.cpp

```diff
@@ -26,16 +31,18 @@ int main(int argc, char* argv[]) try {
 
     pipe.start_chat();
     std::cout << "question:\n";
+
     std::getline(std::cin, prompt);
-    pipe.generate(
-        prompt,
-        ov::genai::image(image),
-        ov::genai::streamer(print_subword)
-    );
+    auto resuls = pipe.generate(prompt,
```

Comment:
results are not used, do we need to store them?

Reply:
removed